lib/revision_table_access_check_filter.js

```diff
@@ -34,7 +34,13 @@ module.exports = function(hyper, req, next, options, specInfo) {
             }
             return hyper.get({
                 uri: new URI(htmlPath),
-                headers: req.headers
+                /* TODO FIXME TEMP
+                 * This removes the cache-control header when requesting the HTML
+                 * for redirect inspection. However, this is a temporary hack in
+                 * order to avoid re-renders for no-cache requests. We need to
+                 * implement a proper header inspection routine.
+                 */
+                headers: Object.assign({}, req.headers, { 'cache-control': undefined })
```

Comment:
There is actually no need to forward the cookie header explicitly. For internal requests it is completely unused, and for external requests in private wikis [it is transparently injected](https://github.com/wikimedia/hyperswitch/blob/87e12a62cf408ba848aa68b247f0f3d7e4d96a7e/lib/filters/http.js#L40-L47).